### DISCUSSION TITLE
通过PCL2的盗号行为

### DISCUSSION THREAD

**Discussion by xisTC** [Meloong-Git/PCL#4711] [提问] [answered]
今天早上，微软给我发了安全信息解绑的邮箱
![Screenshot 2024-09-11 150506](https://github.com/user-attachments/assets/a1ddf734-e487-4143-9820-bbae7e57dfa3)
发现之后立刻更换了密码
![Screenshot 2024-09-11 151052](https://github.com/user-attachments/assets/16ad7448-b6b9-4429-8da7-74c85899a6c1)
改完密码后，我使用微软自带的Windows Security扫描病毒工具脱机扫描，最后扫到两个有害程序。
![Screenshot 2024-09-11 145000](https://github.com/user-attachments/assets/69db935f-5902-48cb-a97f-1a70a16212b3)
![Screenshot 2024-09-11 145359](https://github.com/user-attachments/assets/7170b164-e385-44d5-b7ec-f80732dbc2e6)
最后Security工具帮我把PCL程序移除了。
我有几个问题：
1.这个是下载mod导致的吗？
我下载mod的途径大部分都是PCL自带的下载和MC百科，其他的途径就下载过一两次比如CurseForge和B站评论区分享的网盘。
2.有办法找出盗号程序的位置吗？
3.根据Security指出位置，PCL程序是被感染了吗？
4.我查看了文件夹，发现PCL被移除了，但是mod却依旧存在，病毒mod有被移除吗？

**Comment by tangge233** [comment]
抱歉，刚才回答错了。
“PUA:Win32/GameHack 是 Microsoft Windows Defender 等安全程序使用的术语，用于识别可能导致计算机出现问题的不需要的应用程序。 尽管这些应用程序本身可能无害，但它们仍然会给您的系统带来问题并损害您的隐私。”
并不是说是病毒，所以您的 Microsoft 账号被盗窃与 PCL2 没有关系。

**Reply by xisTC** [reply]
意思是Windows Security把PCL进行了移除，移除错了吗？病毒还在电脑上吗？

**Reply by xisTC** [reply]
安全信息的更换邮件变成了m9world9j20@minecraft.fit这个邮箱，但是我玩MC只用过正版登录器和PCL

**Reply by tangge233** [reply]
你可以对比一下文件信息 2.8.6 的程序哈希值如下
Size: 3288904 bytes
File Version: 2.8.6.0
MD5: 297DC620BABE07B82A08DF3C92CAEB17
SHA1: B2333E2EFEC7D1448279E5DF311D964855E7F153
CRC32: 0EEB6E1A

以及[最新版本的下载地址](https://ltcat.lanzouv.com/b0aj6gsid) 密码：PCL2

**Reply by tangge233** [reply]
> 安全信息的更换邮件变成了某邮箱这个邮箱，但是我玩MC只用过正版登录器和PCL

账号被盗的原因很多，PCL 主程序是不会盗窃你的账号的……看看是不是有病毒、网络环境不安全还是用微软账号登陆过什么网站，如果账号的密码太弱也是有被盗号的可能的。我只能帮你到这了……

**Comment by allMagicNB** [comment]
盲猜是你密码强度太弱，被别人字典碰撞上了。

**Comment by MoYuan-CN** [comment, marked as answer]
这是近期频繁发生的 Microsoft 账户换绑行为，表现为尝试替换成一个 `xxx@minecraft.fit` 邮箱
请检查你的网络环境是否安全，你的计算机中是否有病毒，或者是否使用微软账户登录过非 Microsoft 授权网页

参考：[BV1Rw4m1r78C](https://www.bilibili.com/video/BV1Rw4m1r78C) 等

**Comment by Chiloven945** [comment]
个人建议为了提高账号安全性，可以绑定其他邮箱或手机号码、添加 PIN 或 添加 2FA 等验证方式。
